epub33/rs/index.html

```diff
@@ -2132,6 +2089,10 @@ <h3>Substantive changes since <a href="https://www.w3.org/publishing/epub/epub-s
 					3.2</a></h3>
 
 				<ul>
+					<li>24-Dec-2020: The creation of a <a
+							href="https://www.w3.org/publishing/epub32/epub-packages.html#sec-metadata-elem-identifiers-pid"
+							>release identifier</a> [[EPUB-Packages-32]] is no longer defined. See <a
+							href="https://github.com/w3c/publ-epub-revision/issues/1440">issue 1440</a>.</li>
```

Comment:
The generated reference list (done by respec) does not find the EPUB-Packages-32 entry :-(

Reply:
Added one too many hyphens. Fixed now.